tests/system/test_apikey_cmd.py

```diff
@@ -16,11 +65,12 @@ def config(self):
         }
 
     def setUp(self):
-        super(APIKeyBaseTest, self).setUp()
+        super(APIKeyCommandBaseTest, self).setUp()
         self.user = os.getenv("ES_USER", "apm_server_user")
         password = os.getenv("ES_PASS", "changeme")
         self.es_url = self.get_elasticsearch_url(self.user, password)
         self.kibana_url = self.get_kibana_url()
+        self.base = APIKeyBase(self.es_url)
```

Comment:
this is an uncommon pattern, a comment explaining why is needed would help

Reply:
It is using _composition over inheritance_ to avoid multiple inheritance. Happy to add a comment, and change the variable name as I can see how _base_ might be misleading.